src/cosalib/cmdlib.py

```diff
@@ -196,100 +202,3 @@ def get_basearch():
     except AttributeError:
         get_basearch.saved = RpmOstree.get_basearch()
         return get_basearch.saved
-
```

Comment:
Hmm, shouldn't this diff hunk be part of the next commit, so that the `Builds` move happens in the same commit?

Reply:
I think we should just squash it all into a single commit since they're pretty entangled.

If ashcrow wants to keep the attribution for darkmuggle's commits, I think the best approach is to use the [Co-authored-by tag](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors).

Reply:
SGTM. I'll use `Co-authored-by` and squash things done once I've worked out the last remaining gotchas.

Reply:
Hmm interesting. Wonder why they didn't match the [`Co-developed-by`](https://www.kernel.org/doc/html/latest/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by) from kernel dev.

Agreed we should probably squash a bunch of them though if they're not easily split!

Reply:
Squashed down to a few commits.